packages/block-editor/src/components/navigable-toolbar/index.js

```diff
@@ -169,7 +170,9 @@ function useToolbarFocus( {
 		};
 	}, [ initialIndex, initialFocusOnMount, onIndexChange, toolbarRef ] );
 
-	const { getLastFocus } = useSelect( blockEditorStore );
+	const getLastFocus = useSelect(
+		( select ) => unlock( select( blockEditorStore ) ).getLastFocus
+	);
```

Comment:
I think `const { getLastFocus } = unlock( useSelect( blockEditorStore ) );` is supported